main.js

```diff
@@ -1,22 +1,39 @@
-// Defining text characters for the empty and full hearts for you to use later.
 const EMPTY_HEART = '♡'
 const FULL_HEART = '♥'
+const hearts = document.querySelectorAll('.like-glyph');
 
-// Your JavaScript code goes here!
 
+const toggleHeart = heart => {
+  const mySoul = heart.textContent === EMPTY_HEART;
+  heart.textContent = mySoul ? FULL_HEART : EMPTY_HEART;
+  heart.classList.toggle('activated-heart', mySoul);
+};
 
+const maydayMessage = document.querySelector('#modal-message');
+const timeoutStove = 2500;
```

Comment:
While these names are funny, and I do enjoy then, let's keep the naming more descriptive and less whimsical.